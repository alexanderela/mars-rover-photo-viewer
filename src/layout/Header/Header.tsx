import Grid from "@mui/material/Grid2";

export const Header = () => {
  return (
    <Grid
      size={12}
      sx={{
        height: "7rem",
      }}
    ></Grid>
  );
};
