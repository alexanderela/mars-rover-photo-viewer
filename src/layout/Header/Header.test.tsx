import { describe, expect, it } from "vitest";
import { Header } from "./Header";
import { render, screen } from "../../tests/test-utils";
import { MemoryRouter } from "react-router-dom";

describe("<Header />", () => {
  it("renders Header component", () => {
    render(
      <MemoryRouter>
        <Header />
      </MemoryRouter>
    );
    const element = screen.getByText(/Mars Rover Photo Viewer/i);
    expect(element).toBeInTheDocument()
    expect(element).toMatchSnapshot();
  });
});