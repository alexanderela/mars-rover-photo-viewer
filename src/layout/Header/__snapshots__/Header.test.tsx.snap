// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<Header /> > renders Header component 1`] = `
<div
  class="MuiGrid2-root MuiGrid2-direction-xs-row MuiGrid2-grid-xs-12 css-1obpxxd-MuiGrid2-root"
  data-testid="Header"
>
  <h5
    class="MuiTypography-root MuiTypography-h5 css-qtfspc-MuiTypography-root"
  >
    Mars Rover Photo Viewer
  </h5>
  <div
    class="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault css-1ssg4ni-MuiAvatar-root"
  >
    U
  </div>
</div>
`;
